### PR TITLE
GeoJSON endpoint

### PR DESCRIPTION
**Pull request by olafveerman**
Adds a geojson endpoint to support: https://github.com/WorldBank-Transport/rra-frontend/pull/116

This is an example feature of a scenario with 1 POI type (schools), and 2 population counts per origin (total population and female population). The ETA and population counts are stored straight in the `properties` object to be able to support data driven styling in `mapbox-gl`.

``` json
{
  "type": "Feature",
  "properties": {
    "id": 2000016,
    "n": "Arauá",
    "pop": [
      "total",
      "female"
    ],
    "poi": [
      "school"
    ],
    "e0": 754,
    "p0": 11980,
    "pn0": 0.24,
    "p1": 8567,
    "pn1": 1
  },
  "geometry": {
    "type": "Point",
    "coordinates": [
      -37.6181058,
      -11.2644014
    ]
  }
}
```

### CODE REVIEW

**Comment by olafveerman** [comment]
To reduce size of the response further, this endpoint should return something along the lines of:

```
{
  "meta": {
    "pop": [
      "total",
      "female"
    ],
    "poi": [
      "school"
    ],
    "popMaxValues": [ 100000, 75000 ],
    //...
  },
  "results": [
    {
      "id": 2000016,
      "n": "Arauá",
      "e0": 754,
      "p0": 11980,
      "pn0": 0.24,
      "p1": 8567,
      "pn1": 1,
      "coordinates": [ -37.6181058, -11.2644014 ]
    }
  ]
}
```

The GeoJSON can be generated client-side

**Comment by olafveerman** [comment]
Now returns something along the lines below, saving ~60%

``` json
{
  "meta": {
    "poi_type": [
      "pointOfInterest"
    ],
    "pop_type": [
      "population",
      "female"
    ],
    "maxPop": [
      48733,
      12
    ]
  },
  "results": [
    {
      "i": 2000016,
      "n": "Arauá",
      "e-0": 754,
      "p-0": 11980,
      "c": [
        -37.6181,
        -11.2644
      ]
    },
    {
      "i": 2000012,
      "n": "Boquim",
      "e-0": 116,
      "p-0": 25190,
      "c": [
        -37.62042,
        -11.14501
      ]
    }
  ]
}
```

File: app/routes/scenario--results.js [hunk 1]
```diff
@@ -274,5 +274,108 @@ export default [
...
+ * visualization of the results on the map. GeoJSON is built client-side, so
+ * the response is kept to a minimum.
+ * To support data driven styling with mapbox-gl, properties are stored as flat
+ * k:v pairs, instead of nesting them in another object.
+ */
+function mergeOriginETA (results) {
+  let metaData = {
+    'poi_type': [],
+    'pop_type': [],
+    'maxPop': []
```
Comment (danielfdsilva) [inline app/routes/scenario--results.js]: Could we use a consistent case? Either `camelCase` or `snake_case`?

File: test/utils/data.js [hunk 2]
```diff
@@ -752,7 +752,66 @@ export function project2000 () {
...
+      {
+        'id': 3,
+        'project_id': 2000,
+        'scenario_id': 2000,
+        'origin_id': 200003,
+        'project_aa_id': 200001
+      }
+    ]
+  ))
+  .then(() => scenarioResultsPOI(
```
Comment (danielfdsilva) [inline test/utils/data.js]: - The files the analysis produces should be added as well.
- I've been using ids related to the scenario/project content belongs to, that's why they're all similar. Results ids should follow the same pattern.

Comment (olafveerman) [inline test/utils/data.js]: @danielfdsilva 
This is how the results are written to the db right now:
image

Comment (olafveerman) [inline test/utils/data.js]: All comments were addressed

**Comment by olafveerman** [comment]
Comments in the review were addressed. @danielfdsilva

**Pull request merged**